### DISCUSSION TITLE
Cannot Access Project After I Created It

### DISCUSSION THREAD

**Discussion by aaronksaunders** [AntlerVC/firetable#868] [Support / Q&A] [answered]
Following the instruction to connect to an existing firebase project, that I have access to, and after creating the project... i get an error stating that I don't have access to the project I just created.

I can access the project through firebase console, but not through rowy

**Comment by shamsmosowi** [comment, marked as answer]
Hey @aaronksaunders , is this issue still persisting?

**Reply by BeeBombshell** [reply]
Hey guys, closing this discussion since it has been unattended for a long time. @aaronksaunders if you're still facing this issue, feel free to create a new one.
